common/src/main/java/com/wynntils/models/emeralds/EmeraldModel.java

```diff
@@ -38,14 +39,14 @@ public final class EmeraldModel extends Model {
     private static final Pattern M_PATTERN = Pattern.compile("(\\.?\\d+\\.?\\d*)\\s*(m|million)");
     private static final Pattern E_PATTERN = Pattern.compile("(\\d+)($|\\s|\\s*e|\\s*em)(?![^\\d\\s-])");
     private static final Pattern RAW_PRICE_PATTERN = Pattern.compile("\\d+");
-    public static final double TAX_AMOUNT = 1.05;
+    public static final double TAX_AMOUNT = Models.Character.isSilverbullSubscriber() ? 1.03 : 1.05;
```

Comment:
Not really the way to do this. Since this is static final, referencing this class earlier than character model runs could break this. I suggest adding the tax values as private constants, and adding a getter. That will always work.